dev-python/confuse/confuse-1.0.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```

Comment:
Could you remove 2.7 or make proper deps, say, for sphinx? It's impossible to handle without manual work.

Reply:
Done :)